qutebrowser/completion/models/histcategory.py

```diff
@@ -67,6 +72,15 @@ def set_pattern(self, pattern):
         Args:
             pattern: string pattern to filter by.
         """
+        raw_pattern = pattern
```

Comment:
What's the point of this alias, especially because all this happens before modifying `pattern`?

Reply:
raw_pattern is used later on, and I wanted to make it more explicit that the logic below operates on the unchanged 'const' pattern rather than the pattern we overwrite later on (a change breaking this by mistake would lead to a very hard-to-debug issues). I could move the alias below the if if you want and use pattern here instead, but I don't think it really makes a difference.